src/extra/CoolComponent1.spec.js

```diff
@@ -1,19 +0,0 @@
-import React from 'react';
```

Comment:
I think this test should be here. This is the only test he ran
<img width="574" alt="image 2016-12-01 at 10 43 34 am" src="https://cloud.githubusercontent.com/assets/6188161/20807254/076d2e82-b7b3-11e6-86ec-7157f5127cbb.png">